mllm/backends/ascend/memory/AscendMemoryPool.cpp

```
@@ -0,0 +1,106 @@
+
+#include <atb/types.h>
+#include <acl/acl.h>
+#include "AscendMemoryPool.hpp"
+#include "mllm/utils/Common.hpp"
+#include "mllm/utils/Log.hpp"
+
+namespace mllm::ascend {
+
+constexpr size_t POOL_SIZE = 104857600;  // 100 MiB，
+
+AscendMemoryPool::AscendMemoryPool(size_t pool_size = POOL_SIZE) {
```

Clang (14.0.6)

[error] constructor does not initialize these fields: block_mutex_, used_blocks_, free_blocks_

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)